### ISSUE TITLE
Exception raised when "VSCode" is used as $EDITOR.

### ISSUE THREAD

**Issue by KelSolaar** [tfeldmann/organize#24]
Hi,

Title says it all, my editor is VCode, i.e. `code --wait --new-window ` and an exception is raised when issuing `organize config`:

```python
(python-3.6) Kali:organize kelsolaar$ organize config
Traceback (most recent call last):
  File "/usr/local/bin/organize", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/site-packages/organize/main.py", line 95, in main
    config_edit()
  File "/usr/local/lib/python3.6/site-packages/organize/main.py", line 121, in config_edit
    subprocess.call([editor, str(config_path)])
  File "/usr/local/Cellar/python/3.6.4_4/Frameworks/Python.framework/Versions/3.6/lib/python3.6/subprocess.py", line 267, in call
    with Popen(*popenargs, **kwargs) as p:
  File "/usr/local/Cellar/python/3.6.4_4/Frameworks/Python.framework/Versions/3.6/lib/python3.6/subprocess.py", line 709, in __init__
    restore_signals, start_new_session)
  File "/usr/local/Cellar/python/3.6.4_4/Frameworks/Python.framework/Versions/3.6/lib/python3.6/subprocess.py", line 1344, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'code --wait --new-window ': 'code --wait --new-window '
```

Cheers,

Thomas

**Comment by tfeldmann** [comment]
Thank you for reporting. I found it fails when ``$EDITOR`` contains command line arguments. Should be fixed in the next version!

**Comment by tfeldmann** [comment]
I just released v1.4.2 where this should be fixed.
